.envs/.production/.django

```diff
@@ -54,6 +54,9 @@ JIRA_PASSWORD=
 JIRA_CELL_ROLE_ACCOUNT=
 JIRA_CELL_ROLES=
 
+# Sprint
+SPRINT_HOURS_RESERVED_FOR_REVIEW=
```

Comment:
This is going to break the environment when `SPRINT_HOURS_RESERVED_FOR_REVIEW` is not defined in envs.
```suggestion
# SPRINT_HOURS_RESERVED_FOR_REVIEW=
```